src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/TransformationToolbar.java

```diff
@@ -31,89 +32,44 @@
+    WaitHelper.waitForPageToLoad();
+    WaitHelper.waitForElementToBeDisplayed(Helper.locateElementByTestId("transformations-toolbar-container"));
+    Assert.assertTrue(Helper.isElementExists(Helper.getCssSelectorByDataTestId("transformations-toolbar-container")));
   }
-  @Then("Verify if all icons are displayed on Toolbar with \\\"(.*)\\\"")
+  @Then("Verify if all icons are displayed on Toolbar with {string}")
   public void verifyAllTheIconsAreDisplayedOnToolbar(String testId) {
-    try {
-      WaitHelper.waitForPageToLoad();
-      WaitHelper.waitForElementToBeDisplayed(Helper.locateElementByTestId("toolbar-icon-"
-              + testId));
-      Assert.assertTrue(Helper.isElementExists(Helper.getCssSelectorByDataTestId("toolbar-icon-"
-                    + testId)));
-    } catch (Exception e) {
-      System.err.println("error:" + e);
-    }
+    WaitHelper.waitForPageToLoad();
+    WaitHelper.waitForElementToBeDisplayed(Helper.locateElementByTestId("toolbar-icon-"
+        + testId));
+    Assert.assertTrue(Helper.isElementExists(Helper.getCssSelectorByDataTestId("toolbar-icon-"
+        + testId)));
   }
-  @Then("Click on the function names toggle with testId as \\\"(.*)\\\" and \\\"(.*)\\\"")
+  @Then("Click on the function names toggle with testId as {string} and {string}")
   public void clickOnTheSliderButton(String testId, String iconLabelName) {
-    try {
-      Helper.waitSeconds(5);
-      WaitHelper.waitForElementToBeEnabled
-              (Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler"));
-      WebElement toggle = Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler");
-      WaitHelper.waitForElementToBeDisplayed(toggle);
-      WebElement ele = Helper.locateElementByXPath
-        ("//input[@data-testid='transformations-toolbar-icons-function-name-toggler']");
-      Actions action = new Actions(SeleniumDriver.getDriver());
-      WaitHelper.waitForPageToLoad();
-      action.moveToElement(ele).perform();
-      ele.click();
-      WaitHelper.waitForPageToLoad();
-      WebElement icon = Helper.locateElementByTestId("toolbar-icon-title-" + testId);
-      String text = icon.getText();
-      String actual = iconLabelName;
-      Assert.assertEquals(text, actual);
-    } catch (Exception e) {
-      System.err.println("error: " + e);
-    }
+    Helper.waitSeconds(5);
+    WaitHelper.waitForElementToBeEnabled
+    (Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler"));
+    WebElement toggle = Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler");
+    WaitHelper.waitForElementToBeDisplayed(toggle);
+    WebElement ele = Helper.locateElementByXPath
+    ("//input[@data-testid='transformations-toolbar-icons-function-name-toggler']");
+    Actions action = new Actions(SeleniumDriver.getDriver());
+    action.moveToElement(ele).perform();
+    ele.click();
+    WaitHelper.waitForPageToLoad();
```

Comment:
remove

Reply:
Done

Reply:
we have removed `WaitHelper.waitForPageToLoad()`
going forward, we will be using this method only when we are waiting for the page to load